x-pack/legacy/plugins/siem/public/apps/plugin.tsx

```diff
@@ -25,7 +32,11 @@ export class Plugin {
     this.chrome = chrome;
   }
 
-  public start(core: LegacyCoreStart, plugins: PluginsStart) {
+  public start(core: StartCore, plugins: StartPlugins) {
+    // TODO(rylnd): once we're on NP, we can populate version from env.packageInfo
+    core.uiSettings.set(DEFAULT_KBN_VERSION, '8.0.0');
+    core.uiSettings.set(DEFAULT_TIMEZONE_BROWSER, 'UTC');
+
     // @ts-ignore improper type description
     this.chrome.setRootTemplate(template);
     const checkForRoot = () => {
```

Comment:
I believe you can replace all this code with the NP mounting mechanism in `setup`: `core.application.register`